Dockerfile

```diff
@@ -14,32 +14,18 @@
 # limitations under the License.
 #
 
-# This file will work as is for local development. No need to use Dockerfile.build
-
-#build stage
 ARG BASE=golang:1.12-alpine
 FROM ${BASE} AS builder
 
-ARG ALPINE_PKG_BASE="make git gcc libc-dev libsodium-dev zeromq-dev"
-ARG ALPINE_PKG_EXTRA=""
-
-LABEL license='SPDX-License-Identifier: Apache-2.0' \
-  copyright='Copyright (c) 2019: Intel'
-RUN sed -e 's/dl-cdn[.]alpinelinux.org/nl.alpinelinux.org/g' -i~ /etc/apk/repositories
-RUN apk add --no-cache ${ALPINE_PKG_BASE} ${ALPINE_PKG_EXTRA}
-WORKDIR /app
+ARG MAKE='make build'
 
-COPY go.mod .
-RUN go mod download
+RUN $MAKE
```

Comment:
I removed this argument in other build. Doesn't seem to add much.

Reply:
Good catch we should keep this in the Dockerfile. We will need to port to device-camera-go